index.html

```diff
@@ -22,6 +22,10 @@
 
 <p>A <a href="/guides-template/">template for 18F Guides sites</a> that is also a complete 18F Guides publishing tutorial.</p>
 
+<h2>18F Slides</h2>
```

Comment:
Eh, the main Pages landing page itself has no strong conventions at this point. I'll go ahead and merge this as-is for now.